continuous_integration/gpuci/environment-3.10.yaml

```diff
@@ -6,7 +6,7 @@ channels:
 - nvidia
 - nodefaults
 dependencies:
-- c-compiler
```

Comment:
Based on the conda recipe, seems like we still need a C compiler for compilation - do we need to modify this pinning in some way?